src/main/java/uk/gov/hmcts/reform/hmc/repository/HearingPartyRepository.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.hmc.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.reform.hmc.data.HearingPartyEntity;
+
+@Transactional(propagation = Propagation.REQUIRED)
+@Repository
+public interface HearingPartyRepository extends JpaRepository<HearingPartyEntity, Long> {
+
+    @Query("SELECT techPartyId from HearingPartyEntity where partyReference = :partyRef")
+    Long getTechPartyIdByReference(String partyRef);
+
+    @Query("SELECT partyReference from HearingPartyEntity where techPartyId = :techPartyId")
```

Comment:
This "reverse lookup" is needed to workout, given the values in `party_relationship_details` table, the values tor return in the "GET" response

Reply:
Both native queries now removed, Fixed by @danlysiak in https://github.com/hmcts/hmc-cft-hearing-service/pull/153/commits/50b833069211b0402b67717fae973ed563286907